src/nvim/window.c

```diff
@@ -4644,20 +4610,18 @@ static void win_goto_ver(bool up, long count)
 /// @return      found window
 win_T *win_horz_neighbor(tabpage_T *tp, win_T *wp, bool left, long count)
 {
-  frame_T *fr;
-  frame_T *nfr;
-  frame_T *foundfr;
-
-  foundfr = wp->w_frame;
+  frame_T *foundfr = wp->w_frame;
 
   if (wp->w_floating) {
     return win_valid(prevwin) && !prevwin->w_floating ? prevwin : firstwin;
   }
 
   while (count--) {
+    frame_T *nfr;
+
```

Comment:
Removing this blank line looks more consistent with code above

Reply:
I have a tendency to insert linebreaks if the line below is a comment to enforce what statement the comment is associated with.

```C
int a;
int b;
```

```C
int a;

// comment
int b;
```

but for this case I'll just remove.